### ISSUE TITLE
Display sandboxing permissions

### ISSUE THREAD

**Issue by MayeulC** [flathub/linux-store-frontend#122]
It would be nice to list permissions that are required by an app: access to file-system, D-bus, X11, etc; directly on the website.

**Comment by digitalethics** [comment]
I think this is an extremely important issue and should be prioritized. I'd also hope to have clearly visible information of how Flatpak security is different when running Wayland. Technical writing should be easy to understand and come with clear terminology intended to be read by end-users and not developers.

**Comment by ssokolow** [comment]
This is probably the number-one thing I check when considering installing something through Flatpak and it's very annoying right now to have to keep https://github.com/flathub/ open in another tab, paste over the package name, and then dig into the repo to check `finish-args`.

**Comment by bilelmoussaoui** [comment]
> This is probably the number-one thing I check when considering installing something through Flatpak and it's very annoying right now to have to keep https://github.com/flathub/ open in another tab, paste over the package name, and then dig into the repo to check `finish-args`.
> 
Doing flatpak install x already displays the list of permissions of X. You can also revoke permissions you think the app shouldn't use? need? or add new ones. There's a UI utility called Flatseal for managing those permissions as well.

**Comment by ssokolow** [comment]
Yes, if it didn't, I'd *never* install Flatpak packages... however, I don't feel comfortable with intentionally going through a "Please install this" "Are you sure?" "No" workflow for checking manifests and, if I were any less dedicated to sandboxing things, I'd just stick to using APT.
